dashboard/config/locales/scripts.en.yml

```diff
@@ -10376,6 +10384,18 @@ en:
               name: Crowdsourcing
               description_student: ''
               description_teacher: ''
+            Concept Practice with Minecraft:
+              name: Concept Practice with Minecraft
+            Editing Behaviors:
+              name: Editing Behaviors
+            Virtual Pet with Sprite Lab:
+              name: Virtual Pet with Sprite Lab
+            Build A Project:
+              name: Build A Project
+            Dance Party:
+              name: Dance Party
+            'Conditionals in Minecraft: Voyage Aquatic':
```

Comment:
@nkiruka shooting in the dark, but why are these in quotes when the others aren't?

Reply:
Good question.  Sometimes, yml puts strings in quotes and other times the strings are not in quotes.  [See ref.](https://stackoverflow.com/questions/3790454/in-yaml-how-do-i-break-a-string-over-multiple-lines/21699210#21699210)

Reply:
My question isn't so much about yml. It's more about whether our system for how we use these strings can handle this difference? Could these quotes - which are inconsistent with the other strings - be the cause of the problem with seeding?